.github/workflows/pull-requests.yaml

```diff
@@ -0,0 +1,47 @@
+name: Pull Request
+
+on:
+  [pull_request]
+
+jobs:
+  make-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
This is a good idea; it might speed up our other CI as well.  Added it to the list from our related thread.